content/applications/hr/time_off/allocations.rst

```diff
@@ -74,28 +76,60 @@ information:
-   :align: center
    :alt: A new allocation form with all the fields filled out for the annual two week vacation
          granted to all employees.
 
+Multiple Allocations
+--------------------
+
+When allocating time off, it is common to allocate time to several employees at once. This is done
+using the :guilabel:`Multiple Requests` feature.
+
+To allocate time to multiple employees in a single allocation, navigate to :menuselection:`Time Off
+app --> Management --> Allocations`. Then, click the :icon:`fa-gear` :guilabel:`(gear)` icon in the
+upper-left corner, then click :icon:`fa-users` :guilabel:`Multiple Requests`. This reveals a
+:guilabel:`Multiple Requests` pop-up window.
+
+This form is identical to the :guilabel:`Allocation` form, with one additional field:
+:guilabel:`Mode`. The :guilabel:`Mode` field determines how multiple employees are selected.
+
+Using the drop-down menu, select one of the following :guilabel:`Modes`:
+
+- :guilabel:`By Employee`: This option allows for the selection of multiple individual employees
+  that are unrelated in terms of department, company, or tags. Selecting this reveals an
+  :guilabel:`Employees` field. Select the employees to receive the allocation in the
+  :guilabel:`Employees` field. There is n limit to the amount of employees that can be selected.
+- :guilabel:`By Company`: This option allows for the selection of all employees within a specific
+  company. Selecting this reveals a :guilabel:`Company` field. Select the :guilabel:`Company` to
+  assign the allocation to. Only one company can be assigned in the :guilabel:`Company` field. When
+  a company is selected, *all* employees within the company receive the allocation.
+- :guilabel:`By Department`: This option allows for the selection of all employees within a specific
+  department. Selecting this reveals a :guilabel:`Department` field. Select the
+  :guilabel:`Department` to assign the allocation to. Only one department can be assigned in the
+  :guilabel:`Department` field. When a department is selected, *all* employees within the department
+  receive the allocation.
+- :guilabel:`By Employee Tag`: This option allows for the selection of all employees with a specific
+  tag. Selecting this reveals an :guilabel:`Employee Tag` field. Select the desired
+  :guilabel:`Employee Tag` to select all employees with that tag. Only one tag can be assigned in
+  the :guilabel:`Employee Tag` field. When a tag is selected, *all* employees with that tag receive
+  the allocation.
+
+Next, select the :guilabel:`Time Off Type` using the drop-down menu. Once a :guilabel:`Time Off
+Type` is selected, the placeholder name, 'Allocation Request', changes to the name of the selected
+:guilabel:`Time Off Type`, including the amount of days. Change the name of the allocation, if
+desired.
+
+Fill out the remainder of the :ref:`Multiple Requests form <time_off/allocation-form>`, then click
+:guilabel:`Create Allocaitons` when done.
```

Comment:
```suggestion
:guilabel:`Create Allocations` when done.
```

Reply:
Ugh- this wasn't underlined (so I must not have my spell-checker activated or something). I'll get that fixed- thank!